3600/3/app/models/users.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class User < ActiveRecord::Base
```

Comment:
Add validations at least for user

Reply:
When be done also add some of the validations to the database by migration

Reply:
![Screenshot from 2019-08-02 23-13-39](https://user-images.githubusercontent.com/46228563/62396201-310c3800-b57b-11e9-9f21-56e0c88620d0.png)